src/components/Container/src/Container.vue

```diff
@@ -112,6 +112,20 @@ export default {
 			default: undefined,
 			validator: (color) => colord(color).isValid(),
 		},
+		/**
+		 * Padding of the container
+		 */
+		padding: {
+			type: String,
+			default: '16px 24px',
+			validator: (padding) => {
```

Comment:
I believe we can import `import cssValidator from '@square/maker/utils/css-validator';` and just do `validator: cssValidator('padding'),` here.